sys-kernel/dracut/dracut-047.ebuild

```diff
@@ -0,0 +1,166 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1 eutils linux-info toolchain-funcs systemd
+
+DESCRIPTION="Generic initramfs generation tool"
+HOMEPAGE="https://dracut.wiki.kernel.org"
+SRC_URI="mirror://kernel/linux/utils/boot/${PN}/${P}.tar.xz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+IUSE="debug selinux"
+
+RESTRICT="test"
+
+CDEPEND="virtual/udev
+	virtual/pkgconfig
+	>=sys-apps/kmod-23[tools]
+	"
+RDEPEND="${CDEPEND}
+	app-arch/cpio
+	>=app-shells/bash-4.0:0
+	|| (
+		>=sys-apps/sysvinit-2.87-r3
+		sys-apps/systemd[sysv-utils]
+	)
+	sys-apps/coreutils[xattr(-)]
+	>=sys-apps/util-linux-2.21
+
+	debug? ( dev-util/strace )
+	selinux? (
+		sys-libs/libselinux
+		sys-libs/libsepol
+		sec-policy/selinux-dracut
+	)
+	!net-analyzer/arping
```

Comment:
Weak blockers are for file collisions. Is this really the problem here?

Reply:
No, it is a blocker to prevent dracut from picking incompatible arping implementation. And since arping itself is an optional runtime dependency, this blocker is rather controversial. We should patch dracut itself.
https://bugs.gentoo.org/590566

Reply:
Then please use `!!` strong blocker to make sure users knowingly agrees to removing arping rather than accidentally loses it as a result of installing dracut.